fetch.bs

```diff
@@ -2912,6 +2903,19 @@ reused across <a>connections</a> whose <a for=connection>credentials</a> are fal
 
 <hr>
 
+<div algorithm>
+<p>To <dfn>resolve proxies</dfn>, return the result of finding proxies for <var>url</var> in an
+<a>implementation-defined</a> manner. If there are no proxies, return « "<code>DIRECT</code>" ».
```

Comment:
Where does _url_ come from here? If it needs to be an argument, let's make it explicit.

Reply:
Done